pkgs/development/python-modules/decora_wifi/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, python313Packages }:
+
+buildPythonPackage rec {
+  pname = "decora_wifi";
+  version = "1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "04842cb31129c022328f9cf43767aff5c26bff2f74ee0d0b6fb879883f5d0293";
```

Comment:
Prefer `hash` with SRI-formatted hashes. You can get the SRI hash with `nix hash convert to-sri`.